t/no_pod.t

```diff
@@ -18,14 +18,14 @@ my $main_module_path       = module_path('Test::Pod::CoverageChange');
 
 subtest 'Module with no pod, unexpected' => sub {
     test_out("not ok 1 - Pod coverage on $test_module", "not ok 2 # TODO & SKIP There is no POD in the file $test_module_path.");
-    test_diag("  Failed test 'Pod coverage on $test_module'", "  at $main_module_path line 137.", "$test_module: couldn't find pod");
-    Test::Pod::CoverageChange::pod_coverage_syntax_ok($test_module_path);
```

Comment:
This used to work? Let's make it work again. The sub can do some magic for lone scalars.

Reply:
Hm...maybe not, now that I look at the other PRs. So nevermind :)